vendor/bat-native-ads/src/bat/ads/internal/unittest_base.cc

```diff
@@ -107,6 +107,10 @@ AdsImpl* UnitTestBase::GetAds() const {
   return ads_.get();
 }
 
+CovariateLogs* UnitTestBase::GetCovariateLogs() const {
```

Comment:
Is this needed to be exposed publicly? As we could instantiate covariate logs in the tests.